pyproject.toml

```diff
@@ -27,20 +27,20 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
 cftime = ">=1.5"
 numpy = "*"
 openscm-units = "*"
 packaging = "*"
 pandas = ">=1.1"
-pint = "<0.20"
```

Comment:
I thought that the latest version of pint was incompatible with scmdata without changing a bunch of tests. Was that not true @znicholls ?

Reply:
I think there was a double fix required so pint and pint-pandas (or pandas) got updated, then the whole ecosystem started behaving again